README.md

```diff
@@ -11,6 +11,12 @@ Variable_Name : Integer;
 Variable_With_TLA : Integer;
 ```
 
+## Aspects and Pragmas
+
+Whereever possible aspects should be used. Pragmas should only be used if
+there is no equal aspect or if using the aspect would cause a bug in the
+compiler.
```

Comment:
- We could required a comment pointing to an issue if the the pragma is due to a bug
- compiler -> toolchain (could also be `gnatprove`)

Reply:
What's your opinion on requiring a comment with a workaround in case of a bug?

Reply:
@jklmnn I still think we should require a comment pointing to a workaround in the style guide.

Reply:
The problem is that we don't have an entry for this bug and I don't have a reproducer anymore. Adding the entry to the workarounds repository that we could link to wouldn't add much information.

Reply:
The particular issue wasn't my pointer here. IMHO, we should require in the style guide that you justify the use of a pragma with a comment pointing to some issue.

Reply:
I added a clarification in https://github.com/Componolit/ada-style/pull/8/commits/90aa618b7446346ffe79f8ba50baea712cbac8de.